jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -436,6 +436,8 @@ def _auto_slider_range(self, event=None):
             return
         # if set to auto, default the range based on the limits of the spectrum plot
         x_min, x_max = self.spectrum_viewer_min, self.spectrum_viewer_max
+        if x_min == x_max:
```

Comment:
Given they are floats, might want `allclose` instead of `==`.

Reply:
This is just to prevent a division by zero, so probably is ok?  Depending on the units, I would hate to have `allclose` trigger when there really is an intentional range.  Should I add a comment to this effect or do you think there is a case where `allclose` is safer?

Reply:
Actually, there is a `np.isinf(ndec)` check below, so why do you need `==` here?

Reply:
hmm, I can try to get rid of it, perhaps it was a red herring (I can't remember what test was having issues here).  I guess you could still run into problems if `x_max + x_min == 0` since that would still cause a division by zero when trying to compute `half_range`, but that isn't what this code would cover anyways unless it just happened to be `x_min == x_max == 0` 🤔

Reply:
looking at this closer, I think the existing logic _assumes_ both `x_min` and `x_max` are positive - I'll see if I can generalize that

Reply:
the logic really breaks down near zero because this is getting mapped to redshift.  Since it is only used for the redshift slider bounds, I think the easiest way forward is just to leave the limits as-is in this case, but I did generalize the code to explicitly check for `x_mid == 0` now, instead.